.goreleaser.yaml

```diff
@@ -8,7 +8,7 @@ builds:
   - main: ./hawkwing/
     binary: hawkwing
     ldflags:
-      - -s -w -X main.build={{.Version}}
+      - -s -w -X github.com/hawkv6/hawkwing/internal/version.Version={{.Version}}
```

Comment:
Wouldn't it be enough to use `version.Version={{.Version}}`?

Reply:
No, we need the full import path `github.com/hawkv6/hawkwing/internal/version.Version`. The Go linker's -X flag requires the complete import path to uniquely identify which package variable to set. Using just `version.Version` wouldn't work because the linker needs to know the absolute package location. Would be different if we don't have a compiled version command.